target/linux/brcm63xx/image/bcm63xx.mk

```diff
@@ -489,7 +529,9 @@ TARGET_DEVICES += WAP-5813n
 ### D-Link ###
 define Device/DSL2640B-B
   $(Device/bcm63xx)
-  DEVICE_TITLE := D-Link DSL-2640B rev B2
+  DEVICE_VENDOR := D-Link
+  DEVICE_MODEL := DSL-2640B
+  DEVICE_VARIANT := rev B2
```

Comment:
I would go for only "B2" here. We have the same style for D-Link devices in ramips.